### ISSUE TITLE
When I used the parameters you provided and the pre-trained model, I used the DDSM data set to fine-tune the network, but the AUC value was 0.5. I wonder why the results in your paper could not be reached

### ISSUE THREAD

**Issue by zoupei123456** [nyukat/BIRADS_classifier#10]
I'm using the loss function:cross_entropy = tf.reduce_mean(tf.nn.softmax_cross_entropy_with_logits(labels=y, logits=y_prediction_birads)),The optimizer is :train_step = tf.train.AdamOptimizer(lr).minimize(cross_entropy)
